windowsforms/matching-game/net8-windows/cs/MatchingGame/App.config

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <configSections>
+    </configSections>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8"/>
```

Comment:
Why was this file added under the `net8-windows` configuration?

Reply:
@Tanya-Solyanik This was carried over from using the VS Upgrade plugin. I guess I can delete it. Thanks for spotting that!